.github/workflows/release.yml

```
@@ -0,0 +1,72 @@
+            VITE_PUBLIC_APP_URL=${{ secrets.VITE_PUBLIC_APP_URL || 'http://localhost:8080' }}
+          tags: |
+            ghcr.io/ardakilic/brewform-web:latest
+            ghcr.io/ardakilic/brewform-web:${{ github.sha }}
+            ${{ startsWith(github.ref, 'refs/tags/v') && format('ghcr.io/ardakilic/brewform-web:{0}', github.ref_name) || '' }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: [api, web]
+    if: ${{ secrets.COOLIFY_API_WEBHOOK != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)